### PR TITLE
[Rakefile] Add empty master changelog section on release

### PR DESCRIPTION
**Pull request by segiddins**
@kylef you might want to review this one?

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -735,6 +738,31 @@ def github_access_token_query
...
+    changelog_file = 'CHANGELOG.md'
+    changelog = File.read(changelog_file)
+    return if changelog.match(/^## (.+)/).captures.first == 'Master'
+    return unless current_branch == 'master'
+    # This uses an array of strings so text editors dont strip the trailing spaces
+    changelog.sub! '##', ['## Master',
+                          '',
+                          '##### Enhancements',
+                          '',
+                          '* None.  ',
```
Comment (kylef) [inline Rakefile]: What's the practise here, the first PR to add an enhancement will replace the None?

Comment (kylef) [inline Rakefile]: Seems like an additional reason for multiple PRs to conflict.

Comment (segiddins) [inline Rakefile]: They'll conflict anyways because we like extra newlines between the entries.

**Comment by segiddins** [comment]
@CocoaPods/core can I get a :+1: for this?

**Comment by orta** [comment]
Seems reasonable to me

**Comment by jpsim** [comment]
I couldn't run `rake bootstrap` because of this change:

```
rake bootstrap --trace
rake aborted!
SyntaxError: /Users/jp/Projects/Rainforest/Rakefile:446: syntax error, unexpected tSTRING_BEG, expecting keyword_do or '{' or '('
...aster_changelog_section && sh "git commit -am '[CHANGELOG] A...
...                               ^
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/rake_module.rb:28:in `load'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/rake_module.rb:28:in `load_rakefile'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/application.rb:689:in `raw_load_rakefile'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/application.rb:94:in `block in load_rakefile'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/application.rb:176:in `standard_exception_handling'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/application.rb:93:in `load_rakefile'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/application.rb:77:in `block in run'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/application.rb:176:in `standard_exception_handling'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/2.2.0/rake/application.rb:75:in `run'
/usr/local/opt/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rake-10.4.2/bin/rake:33:in `<top (required)>'
/usr/local/opt/rbenv/versions/2.2.3/bin/rake:23:in `load'
/usr/local/opt/rbenv/versions/2.2.3/bin/rake:23:in `<main>'
```

**Comment by segiddins** [comment]
@jpsim I think just adding parens around the argument to sh on that line should fix it, if you want to make a PR

**Comment by jpsim** [comment]
Thanks for the tip, that did it. I just filed #56.

**Pull request merged**